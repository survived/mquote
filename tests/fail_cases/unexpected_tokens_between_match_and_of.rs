extern crate mquote;

use mquote::mquote;

fn main() {
    let _ = mquote!(
        #{match Some("1")} unexpected stuff
            #{of Some(x)}
                #{x}
            #{of None}
                "default value"
        #{endmatch}
    );
}
