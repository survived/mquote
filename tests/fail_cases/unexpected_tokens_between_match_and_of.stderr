error: Expected #{of ..}
 --> $DIR/unexpected_tokens_between_match_and_of.rs:7:28
  |
7 |         #{match Some("1")} unexpected stuff
  |                            ^^^^^^^^^^
